admin/installer/README.rst

```diff
@@ -0,0 +1,74 @@
+FAQ for Maintainers
```

Comment:
I think we should move the documentation to `docs/gettingstarted/infrastructure`. That's where I've documented the installer tests and the packer AMI builder.